ConsoleGUI/Controls/VerticalScrollPanel.cs

```diff
@@ -105,19 +113,16 @@ void IDrawingContextListener.OnUpdate(DrawingContext drawingContext, Rect rect)
 
 		void IInputListener.OnInput(InputEvent inputEvent)
 		{
-			switch (inputEvent.Key.Key)
+			if (inputEvent.Key.Key == ScrollUpKey)
 			{
-				case ConsoleKey.UpArrow:
-					Top -= 1;
-					return;
-				case ConsoleKey.DownArrow:
-					Top += 1;
-					break;
-				default:
-					return;
+				Top -= 1;
 			}
-
+			else if (inputEvent.Key.Key == ScrollDownKey)
+			{
+				Top += 1;
+			}
+			
 			inputEvent.Handled = true;
 		}
 	}
-}
+}
```

Comment:
I'm not very passionate about new lines at the end of the file or the lack of them, so probably fine to just leave that change in.